lib/Backend/LinearScan.cpp

```diff
@@ -2909,15 +2865,14 @@ LinearScan::Spill(Lifetime *newLifetime, IR::RegOpnd *regOpnd, bool dontSpillCur
 {
     uint minSpillCost = (uint)-1;
 
-    Assert(!newLifetime || !regOpnd || newLifetime->isFloat == (regOpnd->GetType() == TyMachDouble) || newLifetime->isSimd128() == (regOpnd->IsSimd128()));
```

Comment:
The lifetime no longer knows the difference between float and simd, so we can't check